ga4gh/cli.py

```diff
@@ -256,6 +256,9 @@ def addGlobalOptions(parser):
         "--config-file", "-f", type=str, default=None,
         help="The configuration file to use")
     parser.add_argument(
+        "--oidc-config-file", "-o", type=str, default=None,
```

Comment:
Do we need a separate config file for OIDC stuff? Why can't (or shouldn't?) we include all the information we need in the main Flask config file?